aws_cloudwatch/aws_cloudwatch.yml

```diff
@@ -668,3 +667,56 @@ spec:
         yPos: 3
     description: Monitoring AWS/EC2 namespace and specific instance_id
     name: AWS CloudWatch Instance Monitoring
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Telegraf
+metadata:
+    name: thirsty-jang-694005
+spec:
+    associations:
+      - kind: Label
+        name: dreamy-hamilton-294001
+    config: |
+        [agent]
+          debug = true
+          interval = "30s"
+          flush_interval = "10s"
+          metric_buffer_limit = 50000
+        ## sample output, can be replaced by your influxdb servers
+        ## k8s
+        [[outputs.influxdb_v2]]
+          urls = ["http://influxdb.monitoring:9999"]
+          organization = "InfluxData"
+          bucket = "kubernetes"
+          token = "secret-token"
+        ## cloudv2 sample
+        [[outputs.influxdb_v2]]
+          urls = ["https://eu-central-1-1.aws.cloud2.influxdata.com"]
+          token = "$INFLUX_TOKEN"
+          organization = "$INFLUX_CLOUDV2_USERNAME"
```

Comment:
We typically use $INFLUX_ORG for this, unless there's a reason for it to be different